src/signals/incident/services/custom-validators/custom-validators.ts

```diff
@@ -22,10 +22,9 @@ export const validatePhoneNumber = (control?: Control) => {
 }
 
 export const nullOrNumber = (control: Control) => {
-  if (!control || typeof control.value === 'number' || control.value === null) {
+  if (!control || typeof control.value === 'number' || !control.value) {
```

Comment:
Would expect an update in the test so that both `null` and `undefined` are covered.